content/en/docs/resources/guides/extending-jx/plugins.md

```diff
@@ -4,7 +4,8 @@ linktitle: Binary Plugins
 description: Binary Plugins for the Jenkins X CLI
 weight: 10
 aliases:
-    - /docs/contributing/addons/plugins/
+	- /docs/contributing/addons/plugins/
```

Comment:
@MarckK your build failed because of an invalid whitespace here. Please use spaces.